app/src/main/java/com/raygun/raygun4android/sample/MainActivity.kt

```diff
@@ -60,40 +64,52 @@ class MainActivity : AppCompatActivity() {
             } catch (ex: Exception) {
                 val i = 4
                 Log.d("Raygun4Android-Sample", "This is here purely so that our alternative value for i gets used and not optimised away in a release build: $i")
+                Snackbar.make(it, getString(R.string.you_just_created_and_caught_an_exception),Snackbar.LENGTH_SHORT).show()
             }
         }
 
         buttonSetUserAnon.setOnClickListener {
             val user = RaygunUserInfo()
             RaygunClient.setUser(user)
+            Snackbar.make(it, getString(R.string.user_is_now_set_to_anonymous_for_future_raygun_reports),Snackbar.LENGTH_SHORT).show()
```

Comment:
```suggestion
            Snackbar.make(it, getString(R.string.user_is_now_set_to_anonymous_for_future_raygun_reports), Snackbar.LENGTH_SHORT).show()
```

Small thing, just a formatting change. It's also present in the `Snackbar.make` calls.

Reply:
I might leave it as is for now, one of the next things this needs is a linter setup and formatting at build etc anyway.